client/src/lib/components/auction/auction-hud.svelte

```diff
@@ -13,8 +13,8 @@
   let currentTime = $state(Date.now());
 
   let selectedToken = $state<Token | null>(null);
-  let stakeAmount = $state<number>(0);
-  let sellAmount = $state<number>(0);
+  let stakeAmount = $state<number>(100);
```

Comment:
```suggestion
  //TODO: Change defaults values into an error component
  let stakeAmount = $state<number>(100);
```
We want to get this out of the door fast, so no need to fix it rn. I just want to have something greppable when we have the time though